make/autoconf/jdk-options.m4

```diff
@@ -355,6 +355,29 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_DEBUG_SYMBOLS],
   AC_SUBST(SHIP_DEBUG_SYMBOLS)
 ])
 
+# SapMachine 2024-12-05: import JMC agent jars
+AC_DEFUN_ONCE([JDKOPT_SETUP_SAP_JMC_AGENT],
+[
+  AC_ARG_WITH(sap-jmc-agent, [AS_HELP_STRING([--with-sap-jmc-agent],
+      [Set import path for the SAP JMC agent jars])])
+  SAP_JMC_AGENT_ENABLED=false
+  if test "x$with_sap_jmc_agent" != x; then
+    SAP_JMC_AGENT_PATH="$with_sap_jmc_agent"
+	UTIL_FIXUP_PATH(SAP_JMC_AGENT_PATH)
+    if test -f "$SAP_JMC_AGENT_PATH"/agent-*-boot.jar; then
+      SAP_JMC_AGENT_VERSION=`echo "$SAP_JMC_AGENT_PATH"/agent-*-boot.jar | sed -r 's!.*/agent-(.+)-boot.jar!\1!'`
```

Comment:
Usually  $SED is used in the m4 files ,  could you please consider this ? (Also $(ECHO) is mostly used)
Have you checked that the sed-flag  -r works cross platform ?

Reply:
Good catch. Since the -r option is not used anywhere else, I've changed it to just use plain sed features.